modules/dataquery/js/workers/savezip.js

```diff
@@ -1,5 +1,6 @@
 /*global self: false, Blob: false */
-importScripts(Loris.BaseURL + "/js/jszip/jszip.min.js");
+var BaseURL = self.location.href.split(self.location.pathname)[0];
```

Comment:
You are right, the web worker runs in a different window so even global variable are out of reach. I can't really see why this would not work if it extract the baseURL from the actual URL.